mobify/platform-scaffold
# Feature/mbfycnct 27 get cart cart parsers

This PR covers functionality regarding the getCart command and cart parsers


 **JIRA**: ([MBFYCNCT-27](https://thinkwrap.jira.com/browse/MBFYCNCT-27))


## Changes
-  Added getCart command
-  Added parseCatrProducts and parseCartContents to cart/parsers.js
-  Added cart utilities to cart/utils.js, including fetchCartItemData method
-  Fixed addToCart command which wasn’t showing the add to cart popup when product was added
-  Check in updateItemQuantity and addToCart commands if product was successfully added/updated 

## Todos
- [ ] Review why only 1 item is added to the cart from PDP even if higher quantity is selected

## How to test-drive this PR
- Run `npm run dev`
- Preview [Thinkwrap accelerator](https://preview.mobify.com/?url=https%3A%2F%2Fhydemo-electronics.thinkwrap.com%2Fyacceleratorstorefront%2Felectronics%2Fen%2F%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Navigate to a PDP as an anonymous user and add the product to the cart
- Login wth user: alistair@hybris.com and password: 123456
- Verify cart was merged correctly
- Update items in the cart (quantity) and remove items
